app/assets/stylesheets/govuk_publishing_components/components/_previous-and-next-navigation.scss

```diff
@@ -11,12 +11,14 @@
 .gem-c-pagination__item {
   @include govuk-font($size: 16, $line-height: (20 / 16));
   list-style: none;
+  border-top: 1px solid $govuk-border-colour;
```

Comment:
I'm not convinced about this border, specifically when the component is used in search results (see below) as there's a repetition of a border from the results into the next/prev. Not sure what to suggest, though. What's the intended purpose of the border?

<img width="619" alt="Screenshot 2020-02-24 at 15 51 46" src="https://user-images.githubusercontent.com/861310/75167772-cb9db580-571d-11ea-82b3-daedd6ff7466.png">

Reply:
will run comment past designer

Reply:
The design looks like this on purpose. The pagination as it currently appears is just 'floating' and to my mind looks a bit lost/ strange. I've added the border to contain it.